app/webpacker/components/Results/resultsFilter.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react';
+import {
+  Button, ButtonGroup, Form, Segment,
+} from 'semantic-ui-react';
+import { EventSelector } from '../wca/EventSelector';
+import { RegionSelector } from '../CompetitionsOverview/CompetitionsFilters';
+import { countries } from '../../lib/wca-data.js.erb';
+import I18n from '../../lib/i18n';
+
+export default function ResultsFilter({ filterState }) {
+  const {
+    event,
+    setEvent,
+    region,
+    setRegion,
+    rankingType,
+    setRankingType,
+    gender,
+    setGender,
+    show,
+    setShow,
+  } = filterState;
+  const regionIso2 = useMemo(() => {
+    if (region === 'world') {
+      return 'all';
+    }
+    const iso2 = countries.real.find((country) => country.id === region)?.iso2;
+    if (iso2) {
+      return iso2;
+    }
+    return region;
+  }, [region]);
+  return (
+    <Segment raised>
+      <Form>
+        <Form.Field>
+          <EventSelector
+            selectedEvents={[event]}
+            onEventSelection={({ eventId }) => setEvent(eventId)}
+            hideAllButton
+            hideClearButton
+          />
+          <RegionSelector
+            region={regionIso2}
+            dispatchFilter={({ region: r }) => setRegion(countries.byIso2[r]?.id ?? r)}
+          />
+        </Form.Field>
+        <Form.Group>
+          <Form.Field width={3}>
+            <label>{I18n.t('results.selector_elements.type_selector.type')}</label>
+            <ButtonGroup primary>
+              <Button
+                active={rankingType === 'single'}
+                onClick={() => setRankingType('single')}
+              >
+                {I18n.t('results.selector_elements.type_selector.single')}
+              </Button>
+              <Button active={rankingType === 'average'} onClick={() => setRankingType('average')}>{I18n.t('results.selector_elements.type_selector.average')}</Button>
+            </ButtonGroup>
+          </Form.Field>
+          {/* <Form.Field width={1}> */}
+          {/*   <ButtonGroup> */}
+          {/*    <Button>All years</Button> */}
+          {/*   </ButtonGroup> */}
+          {/* </Form.Field> */}
```

Comment:
Remove?

Reply:
I kept this in as a reminder to include it when we bring back the year filter